src/librustc_resolve/resolve_imports.rs

```diff
@@ -1350,8 +1350,8 @@ impl<'a, 'b> ImportResolver<'a, 'b> {
                         }
                     }
                     reexports.push(Export {
-                        ident: ident.modern(),
```

Comment:
To be sure, `modern()` is removed here and above because idents are already kept in "modernized" form in `resolutions`?

Reply:
Correct